strapi/strapi
# Publish the valid selected entries in the modal

### What does it do?

- Publishes the selected entries in the validation modal that are ready to published (or already published). Entries with validation errors will not be sent to the server for publishing

### TODO
- [x] CS-50 Handle the API response (success and error state)

### Why is it needed?

- To improve the bulk publishing experience

### How to test it?

- Select a bunch of entries some that have error and some that don't and open the Modal
- Publish them, you should be left with only the ones that had validation errors

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
